libcudacxx/include/cuda/std/detail/libcxx/include/__type_traits/is_swappable.h

```diff
@@ -84,12 +80,8 @@ struct __swappable_with<_Tp, _Up,  false> : false_type {};
 template <class _Tp, class _Up = _Tp, bool _Swappable = __swappable_with<_Tp, _Up>::value>
 struct __nothrow_swappable_with {
   static const bool value =
-#ifndef _LIBCUDACXX_HAS_NO_NOEXCEPT
```

Comment:
Why is this no longer needed?

Reply:
Ditto, every C++11 compiler has support for noexcept and this is a remnant for libc++ C++03 mode

Reply:
I'd be fine with filing an issue to remove all noexcept wrappers in this case.

Reply:
I have filed https://github.com/NVIDIA/cccl/issues/127